forge.config.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
         authors: 'Electron Community',
         exe: 'electron-fiddle.exe',
         iconUrl: 'https://github.com/electron/electron/blob/fe0f203/atom/browser/resources/win/atom.ico?raw=true',
```

Comment:
Change the `iconUrl` too? Or is that post-PR?

Reply:
Post-PR, just so that we have a stable path.